src/explore-education-statistics-common/src/components/ContentSectionIndex.tsx

```diff
@@ -20,40 +21,49 @@ interface Props {
   visible?: boolean;
 }
 
+interface ListItem {
+  id: string;
+  tagName: string;
+  textContent: string;
+}
+
+export interface ParentListItem extends ListItem {
+  children: Array<ListItem>;
```

Comment:
I think in general we've used the syntax `ListItem[]` over `Array<ListItem>` in the majority of the codebase, unless there's a difference?

Reply:
Yep, will change here and in the other places. Still getting used to Typescript!